src/Core/SettingsChangesHistory.cpp

```diff
@@ -97,6 +97,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"allow_general_join_planning", false, true, "Allow more general join planning algorithm when hash join algorithm is enabled."},
             {"optimize_extract_common_expressions", false, false, "Introduce setting to optimize WHERE, PREWHERE, ON, HAVING and QUALIFY expressions by extracting common expressions out from disjunction of conjunctions."},
             {"max_bytes_ratio_before_external_sort", 0., 0., "New setting."},
+            {"min_external_sort_block_bytes", 0., 100_MiB, "New setting."},
```

Comment:
The default value is very bad, but, it will be removed anyway - https://github.com/ClickHouse/ClickHouse/pull/80777